must-gather/collection-scripts/gather_noobaa_resources

```diff
@@ -60,9 +60,9 @@ done
 # Add important notfications to a notifications.txt file at the root of the noobaa collection path
 UNMANAGED_NOOBAA_NOTIF="The noobaa deployed on this cluster is not managed by the storagecluster CR and will not react to configuration changes made to storagecluster CR"
 
-NOOBAA_RECONCILE_STRATEGY=$( oc -n ${ns} get storagecluster -o jsonpath='{.items[0].spec.multiCloudGateway.reconcileStrategy}' 2> /dev/null)
+NOOBAA_RECONCILE_STRATEGY=$( oc -n "${ns}" get storagecluster -o jsonpath='{.items[0].spec.multiCloudGateway.reconcileStrategy}' 2> /dev/null)
 if [ "${NOOBAA_RECONCILE_STRATEGY}" = "ignore" ] || [ "${NOOBAA_RECONCILE_STRATEGY}" = "standalone" ]; then 
-    if [ "$(oc -n ${ns} get noobaa -o name 2> /dev/null) " ]; then 
-        echo "- ${UNMANAGED_NOOBAA_NOTIF}" >> ${NOOBAA_COLLLECTION_PATH}/notifications.txt
+    if [ "$(oc -n "${ns}" get noobaa -o name 2> /dev/null)" ]; then
+        echo "- ${UNMANAGED_NOOBAA_NOTIF}" >> "${NOOBAA_COLLLECTION_PATH}/notifications.txt"
```

Comment:
```suggestion
        echo "- ${UNMANAGED_NOOBAA_NOTIF}" >> "${NOOBAA_COLLLECTION_PATH}"/notifications.txt
```

Reply:
Why? The string should be the full path.

Reply:
we add "" inverted commas to avoid splitting of words and globbing of the var we use. that will be "${NOOBAA_COLLLECTION_PATH}"

e.g. this shellcheck output
```In ./must-gather/collection-scripts/gather_noobaa_resources line 66:
        echo "- ${UNMANAGED_NOOBAA_NOTIF}" >> ${NOOBAA_COLLLECTION_PATH}/notifications.txt
                                              ^------------------------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
        echo "- ${UNMANAGED_NOOBAA_NOTIF}" >> "${NOOBAA_COLLLECTION_PATH}"/notifications.txt```

Reply:
The full path should to be a string (including the "notifications.txt" part) as such I believe @jarrpa suggestion is correct. 
I think that `shellcheck` is too stupid to understand this case and that the suggested solution will also resolve the `shellcheck` error.